### ISSUE TITLE
Request : add different ratio options for desktop and fullscreen

### ISSUE THREAD

**Issue by SeeDborg** [JosefNemec/Playnite#2079]
I'm using a theme in Fullscreen that uses squares covers (1:1 ratio) and I'd like to use a desktop theme that uses 4:5 covers (with the same covers of course).

Would it be possible to implement 2 options for ratio (in desktop AND fullscreen) ?

**Comment by JosefNemec** [comment]
Are you asking for option to set different aspect ratio between Desktop and Fullscreen modes or to be able to assign different covers for Desktop and Fullscreen?

**Comment by JosefNemec** [comment]
I don't know if this makes sense, since you want to use the same images, surely you would want to keep the same aspect ratio that fits those images, right?

**Comment by SeeDborg** [comment]
No, I like having squares in Fullscreen but not in Desktop (because of the different themes in use). Playnite already crops them prefectly.
For example :
In desktop I have 4:5 ratio covers : https://i.imgur.com/6PbcH16.jpg
In Fullscreen, 1:1 covers : https://i.imgur.com/yLfCTiH.jpeg

**Comment by SeeDborg** [comment]
In fact it seems logical : people don't have the same themes for desktop and fullscreen, so maybe they'd like different ratios for one theme and another, don't you think ?

**Comment by SeeDborg** [comment]
Since your last update, it's better. Now, if my covers pictures are square but I set 4:5 ratio, it's still OK in desktop mode, but I have that in Fullscreen mode :

https://i.imgur.com/0fcsCIu.jpeg

Too much space in the lines ^^
